lib/service.js

```diff
@@ -21,154 +24,67 @@ export default class WdioImageComparisonService extends BaseClass {
      * @param {Array.<Object>} capabilities list of capabilities details
      */
     before(capabilities) {
-        const instanceData = getInstanceData(capabilities)
-        const folders = this.folders
-        const defaultOptions = this.defaultOptions
-
-        /**
-         * Saves an image of an element
-         */
-        browser.addCommand('saveElement', function (element, tag, saveElementOptions = {}) {
-            return saveElement({
-                executor: this.execute.bind(browser),
-                screenShot: this.takeScreenshot.bind(browser),
-            },
-            instanceData,
-            getFolders(saveElementOptions, folders),
-            element,
-            tag,
-            {
-                wic: defaultOptions,
-                method: saveElementOptions,
-            },
-            )
-        })
-
-        /**
-         * Saves an image of a viewport
-         */
-        browser.addCommand('saveScreen', function (tag, saveScreenOptions = {}) {
-            return saveScreen({
-                executor: this.execute.bind(browser),
-                screenShot: this.takeScreenshot.bind(browser),
-            },
-            instanceData,
-            getFolders(saveScreenOptions, folders),
-            tag,
-            {
-                wic: defaultOptions,
-                method: saveScreenOptions,
-            },
-            )
-        })
-
-        /**
-         * Saves an image of the complete screen
-         */
-        browser.addCommand('saveFullPageScreen', function (tag, saveFullPageScreenOptions = {}) {
-            return saveFullPageScreen({
-                executor: this.execute.bind(browser),
-                screenShot: this.takeScreenshot.bind(browser),
-            },
-            instanceData,
-            getFolders(saveFullPageScreenOptions, folders),
-            tag,
-            {
-                wic: defaultOptions,
-                method: saveFullPageScreenOptions,
-            },
-            )
-        })
-
-        /**
-         * Saves an image of the complete screen with the tabbable lines and dots
-         */
-        browser.addCommand('saveTabbablePage', function (tag, saveTabbableOptions = {}) {
-            return saveTabbablePage({
-                executor: this.execute.bind(browser),
-                screenShot: this.takeScreenshot.bind(browser),
-            },
-            instanceData,
-            getFolders(saveTabbableOptions, folders),
-            tag,
-            {
-                wic: defaultOptions,
-                method: saveTabbableOptions,
-            },
-            )
-        })
+        if(typeof capabilities['browserName'] !== 'undefined') {
+            console.log('Adding to global browser')
```

Comment:
Can you add this with the wdio-logger instead of a `console.log()`, by adding this in the file

`npm install -S @wdio/logger`

```js
import logger from '@wdio/logger'

const log = logger('wdio-image-comparison-service')

log.info('Adding to global browser')
```
as seen in here https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-sauce-service/src/service.ts